codeforamerica/vita-min
# FYST-1065 AZ content updates pt. 1

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-1065
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Paired with Anu through a bunch of copy and styling updates; didn't take screenshots of everything because we looked at it together
- There's more to do but wanted to keep the PR short
## How to test?
- Mostly just looked at the pages to make sure they looked right
- Fixed some failing tests that were expecting the old copy
## Screenshots (for visual changes)
- Lots of visual changes but they've already been looked at by design and should be reviewed one more time in acceptance testing
